pkg/constants.go

```diff
@@ -0,0 +1,9 @@
+package constants
```

Comment:
This breaks typical go convention where the package being imported is the same as the name of the directory it's in.

And almost all of these are specific to the auth package, except for `session.json` which is a common file name other commands may need for gaining the session token.

My suggestion:
- move most of these into a `cmd/auth/constants.go` file under the auth package
- move this file to `pkg/constants/constants.go` with the "session.json" variable exposed

There's probably a dozen ways to do this, so if you think there's a cleaner way, feel free to suggest something!